pkg/webhook/policy.go

```diff
@@ -261,6 +261,24 @@ func (h *validationHandler) getValidationMessages(res []*rtypes.Result, req *adm
 				logging.RequestUsername, req.AdmissionRequest.UserInfo.Username,
 			).Info(fmt.Sprintf("denied admission: %s", r.Msg))
 		}
+		if *logEvalMetrics {
```

Comment:
This appears to duplicate logDenies logs. What should the relationship between the two flags be? I don't think `--log-eval-metrics` being a superset of `--log-denies` makes sense.

Either:
* log metrics requires log denies to be supplied to take effect or...
* log metrics exposes a different data set entirely.

There is something else complicating things here:

Evaluation metrics are only shown for violations, this may not be useful for debugging requests that pass validation but the validation is slow.

If we decouple metrics reporting from violations, having metrics be a separate flag makes more sense.